server/migrations/20230127_add_plan_deposit.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex) {
+    return knex.schema.table('plan', table => {
+      table.boolean('was_deposited').default('false');
+    }).then(() => {
+      return knex.raw('UPDATE plan set was_deposited=true where id in (select distinct p.plan_id from plan_status_change p join status_mapping s on s.mavat_status = p.`status` and s.meirim_status in (\'תכנית מבוטלת\',\'תכנית מאושרת\'))');
```

Comment:
@GalGend check status here correct

Reply:
please export to consts

Reply:
const in sql??

Reply:
it's just a string after all, or use the id of the status